EventFilter/SiPixelRawToDigi/python/SiPixelRawToDigiRegional_cfi.py

```diff
@@ -3,7 +3,7 @@
 from EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi import *
 
 ## regional seeded unpacking for specialized HLT paths
-siPixelDigisRegional = siPixelDigis.clone()
+siPixelDigisRegional = siPixelDigis.cpu.clone()
 siPixelDigisRegional.Regions = cms.PSet(
     inputs = cms.VInputTag( "hltL2EtCutDoublePFIsoTau45Trk5" ),
```

Comment:
are you super sure ?

Reply:
I think so, yes - why ?

Reply:
I was not expecting a dependence on the HLT menu

Reply:
this
```
     inputs = cms.VInputTag( "hltL2EtCutDoublePFIsoTau45Trk5" ), 
```
is not something I've added, it's there since before the migration from CVS to Git.

Reply:
,,, and the comment above it says "## regional seeded unpacking for specialized HLT paths"